Google.Cast/component/GettingStarted.md

```diff
@@ -149,6 +149,47 @@ var mediaInformation = new MediaInformation ("http://dummyurl.com/kitten.mp4",
 MediaControlChannel.LoadMedia (mediaInformation, true, 0);
 ```
 
+### Methods with IntPtr type as parameter
```

Comment:
First I think we have a bug with the PlainText attribute and protocols.  Can you file a bug for this with the iOS team?

Rather than document how to deal with IntPtr arguments, I think that we should create an abstract class for the users.

```
public class MyLogger : NSObject, ILoggerDelegate
{
    public void Log(IntPtr function, string message)
    {
        var func = System.Runtime.InteropServices.Marshal.PtrToStringAnsi (function);
        Log (func, message);
    }

    abstract void Log (string function, string message);
}
```